### PR TITLE
Complete login flow and add / delete donation functionality implemented

### PR DESCRIPTION
**Pull request by sebastiancrossa**
This PR adds the finishing touches of the login flow, as well as the ability to load, add, and delete donations.

I also changed up the styling a bit, lmk if you like it. I feel like some of the coloring can change (too much yellow :p). How the UI looks:

* Home screen:
Screen Shot 2020-06-04 at 7 40 02 PM

* User screen:
Screen Shot 2020-06-04 at 7 40 40 PM

### CODE REVIEW

File: src/Auth.js [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
-const Auth = ({ addUser }) => {
-  const authHandler = (error, data) => {
+import { API, graphqlOperation } from "aws-amplify";
+import { createUser } from "./graphql/mutations";
+
+const Auth = ({ addUser, history }) => {
+  const authHandler = async (error, data) => {
     if (error) {
-      alert("Please log into Twitter!");
+      alert(error);
```
Comment (barronwei) [inline src/Auth.js:12]: Could you swap the alert back?

File: src/components/User.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        )
+      );
+
+      console.log(userData.data.listUsers.items);
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  return <>{userData ? children(userData) : children}</>;
```
Comment (barronwei) [inline src/components/User.jsx:33]: Did you mean to add `<div>`?

File: src/Auth.js (hunk 1 above)
Comment (sebastiancrossa) [reply src/Auth.js]: Done!

Just pushed the change onto this same branch. It should appear below as commit [4ff1fe3](https://github.com/MLH-Fellowship/0.2.1-donation-trigger/pull/26/commits/4ff1fe300c532f58b7073badf65a104eecadfcc8)

File: src/components/User.jsx (hunk 2 above)
Comment (sebastiancrossa) [reply src/components/User.jsx:33]: I don't want to create a DOM node, so it's not really needed. A [fragment](https://reactjs.org/docs/fragments.html) seemed like a better fit.

**Pull request merged**